src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1595,15 +1595,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "NewVersionFeatures_Info_about_abb_information" = "Änderungen zum Release finden Sie in den App-Informationen unter dem Menüpunkt „Neue Funktionen”.";
 
-/* Version 2.10 */
+/* Version 2.11 */
 
-"NewVersionFeature_210_boosterVaccination_title" = "Information zu Auffrischungsimpfung";
+"NewVersionFeature_211_universalQRCodeScanner_title" = "Universeller QR-Code-Scanner";
 
-"NewVersionFeature_210_boosterVaccination_description" = "Die App kann Ihnen nun eine Mitteilung anzeigen, die Sie über eine eventuell anstehende Auffrischungsimpfung informiert.";
+"NewVersionFeature_211_universalQRCodeScanner_description" = "Sie können nun unabhängig davon, von wo ein QR-Code-Scanner in der CWA geöffnet wird, alle von der CWA unterstützen Dokumente scannen:\n\t\U2022 PCR-Tests\n\t\U2022 Schnelltests\n\t\U2022 Event Check-ins\n\t\U2022 Zertifikate";
 
-"NewVersionFeature_210_printVersionVaccinationCertificate_title" = "Druckversion von Impfzertifikaten";
+"NewVersionFeature_211_qrCodeImport_title" = "QR-Code importieren";
```

Comment:
@naveeddotio As the feature "QR-Code importieren" will not be delivered with 2.11 for iOS, can you please remove the according bullet point from the text? Figma and JIRA are already adjusted.

Reply:
@ChristinaTsigganos Yes, I am aware about this. It will be adjusted.